server/app/services/rpc_server.rb

```diff
@@ -40,6 +40,10 @@ def handle_request(ws_client, grid_id, message)
       begin
         result = actor.send(method, *message[3])
         send_message(ws_client, [1, msg_id, nil, result])
+        unless actor.alive?
+          error "actor for handler #{handler} did die, removing it from cache"
```

Comment:
Does this ever trigger? If we're the only actor sending it messages, then we should see any exceptions that kill it?